src/utilities/new_versions.jl

```diff
@@ -280,24 +295,69 @@ function make_pr_for_new_version(
             if commit_was_success
                 @info("Commit was a success")
                 api_retry() do
-                    @mock git_push("origin", new_branch_name; force=true, env=env)
+                    @mock git_push(
+                        "origin", new_branch_name, pkey_filename; force=true, env=env
+                    )
                 end
 
-                create_new_pull_request(
+                new_pr, _ = create_new_pull_request(
                     forge,
                     repo,
                     new_branch_name,
                     master_branch_name,
                     new_pr_title,
                     new_pr_body,
                 )
+
+                cc_user && cc_trigger_user(forge, repo, new_pr; env=env)
+                unsub_from_prs && unsub_from_pr(forge, new_pr)
             end
         end
     end
 
     return nothing
 end
 
+function cc_trigger_user(
+    api::GitHub.GitHubAPI, repo::GitHub.Repo, pr::GitHub.PullRequest;
+    env=ENV,
+)
+    username = env["GITHUB_ACTOR"]
+    body = "cc @$username"
+
+    return @mock GitForge.create_pull_request_comment(
+        api,
+        repo.owner.login,
+        repo.name,
+        pr.id;
+        body=body,
+    )
+end
+
+function cc_trigger_user(
+    api::GitLab.GitLabAPI, repo::GitLab.Project, pr::GitLab.MergeRequest;
+    env=ENV,
+)
+    username = env["GITLAB_USER_LOGIN"]
```

Comment:
I wonder if we should add a bunch of env-var helpers to GitForge?

Reply:
I think this is a good idea, and would warrant an issue on GitForge. Also a little bit out of scope imo, but would be nice to set up pre-defined CI ENVVARs and an easy way to grab them without knowing the actual var name, like `e = ForgeCIEnv(::GitLabAPI)` `username(api, e)`, something like that in any case

Reply:
Currently GitForge doesn't necessarily concern itself with CI's, but it probably should